eventuate-tram-view-support-rebuild/src/main/java/io/eventuate/tram/viewsupport/rebuild/TopicPartitionOffsetMessageId.java

```diff
@@ -37,4 +39,12 @@ public long getOffset() {
   public void setOffset(long offset) {
     this.offset = offset;
   }
+
+  public String getDatabaseId() {
+    return messageId;
+  }
+
+  public void setDatabaseId(String databaseId) {
```

Comment:
I really don't like mutable objects like this. Especially since it came from the CDC.

Reply:
ok, I will use make it immutable